packages/bootstrap/scss/prompt/_variables.scss

```diff
@@ -0,0 +1,66 @@
+// Prompt
+
+/// The text color of the of the Prompt.
```

Comment:
There are some redundant words in the variable description that should be omitted: 

The text color of the <ins>of the</ins>, The background color of the <ins>of the</ins>, The border color of the <ins>of the</ins>

_Note:_ This appears from line 3 through 26